app/src/main/java/space/taran/arkmemo/data/repositories/TextNotesRepository.kt

```diff
@@ -88,33 +178,29 @@ class TextNotesRepository @Inject constructor() {
         }
         if (path != null) {
             val file = path.toFile()
-            val noteFile = File(file, "${DUMMY_FILENAME}.${NOTE_EXT}")
-            if (!noteFile.exists()) {
-                try {
-                    val fileWriter = FileWriter(noteFile)
-                    val bufferedWriter = BufferedWriter(fileWriter)
-                    writeToFile(bufferedWriter)
-
-                    val id = computeId(Files.size(noteFile.toPath()), noteFile.toPath())
-
-                    Log.d("Filename", noteFile.name)
-
-                    with(noteFile){
-                        val newFile = File(file, "$id.${NOTE_EXT}")
-
-                        if(!newFile.exists())
-                            if (renameTo(newFile))
-                                Log.d("New filename", newFile.name)
-                            else
-                                removeFileFromMemory(noteFile.toPath())
-
-                    }
-
-                } catch (e: Exception) {
-                    e.printStackTrace()
+            val tempFile = File(file, "${DUMMY_FILENAME}.${NOTE_EXT}")
```

Comment:
_(Future work)_

Probably something standard should be used for creating temporary file.